azureadps-2.0-preview/AzureAD/Get-AzureADAuditSignInLogs.md

```diff
@@ -22,41 +22,41 @@ The Get-AzureADAuditSignInLogs cmdlet gets an Azure Active Directory sign in log
 ## EXAMPLES
 
 ### Example 1: Get sign in logs after a certain date
-```
-PS C:\>Get-AzureADAuditSignInLogs -Filter "createdDateTime gt 2019-03-20"
+```powershell
+PS> Get-AzureADAuditSignInLogs -Filter "createdDateTime -gt 2019-03-20"
```

Comment:
Hello @doctordns, thanks for your contribution. When the - is added before the operator the example fails with 'Invalid filter clause' error. Works alright when the - is removed.
```suggestion
PS> Get-AzureADAuditSignInLogs -Filter "createdDateTime gt 2019-03-20"
```